ansible/README.md

```diff
@@ -308,21 +308,14 @@ without spaces as there is no current fix available to this problem.
 The default system throttling limits are configured in this file [./group_vars/all](./group_vars/all).
```

Comment:
suggested change:

-The default system throttling limits are configured in this file [./group_vars/all](./group_vars/all).
+The default system throttling limits are configured in this file [./group_vars/all](./group_vars/all) and may be changed by modifying the `group_vars` for your specific environment.

Reply:
Looks good to me, thanks!